demux/utils/novaseq_samplesheet.py

```diff
@@ -185,26 +207,32 @@ def pad_and_rc_indexes(
 
         return index1, index2
 
-    def get_reagent_kit_version(self) -> int:
+    def get_reagent_kit_version(self) -> float:
         """ Derives the reagent kit version from the run parameters """
 
         parameter_to_version = {"1": 1.0, "3": 1.5}
```

Comment:
Should we perhaps have this in constants so that it is easier to set or fix if updated or altered?

Reply:
good idea, fixed in https://github.com/Clinical-Genomics/demultiplexing/pull/146/commits/e61c4a043c4e1be96d64b0f18e2a15725abf2056